demo/index.html

```diff
@@ -257,6 +257,10 @@ <h1>Shaka Player <span id="version"></span></h1>
         <div>
           <button id="storeDeleteButton">Store</button>
           <span id="storeDeleteHelpText"></span>
+          <button id="pauseBtn">Pause</button>
```

Comment:
Please change the names to pauseDownloadButton/resumeDownloadButton or similar (I assume that's what they do, right?). pause/resume button sounds like it's pausing/resuming the video.

Reply:
Yes :)